dependency_links.txt

```diff
@@ -1,2 +1 @@
 https://github.com/SciLifeLab/statusdb/archive/master.zip#egg=statusdb
```

Comment:
I think that's not needed wither? or is it @senthil10 no I'm not sure... was it used in storage cleanup for UPPMAX maybe?